### PR TITLE
[new release] bigarray-overlap (0.1.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Bigarray.overlap

- Project page: <a href="https://github.com/dinosaure/overlap">https://github.com/dinosaure/overlap</a>
- Documentation: <a href="https://dinosaure.github.io/overlap/">https://dinosaure.github.io/overlap/</a>

##### CHANGES:

- First release

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
According to your `dune-project` file, `dune >= 2.3` is required. Could you tell me if that's right? Otherwise it looks good

**Comment by dinosaure** [comment]
It's may be too high but it's ok for me.

**Comment by kit-ty-kate** [comment]
Thanks!

File: packages/bigarray-overlap/bigarray-overlap.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "sh" "-exc" "echo \"xen_linkopts = \\\"-l:stubs/xen/liboverlap_stubs_xen_stubs.a\\\"\" >> _build/default/META.overlap"]
+  [ "sh" "-exc" "echo \"freestanding_linkopts = \\\"-l:stubs/freestanding/liboverlap_stubs_freestanding_stubs.a\\\"\" >> _build/default/META.overlap"]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+install: [
+  [ "dune" "install" "-p" name ] {with-test}
+  [ "./test/test_runes.ml" ] {with-test}
+]
```
Comment (kit-ty-kate) [inline packages/bigarray-overlap/bigarray-overlap.0.1.0/opam:23]: why is there a special install field for tests? Especially, what is the purpose of the `dune install`?

**Pull request merged**